.eslintrc.test.js

```diff
@@ -3,6 +3,8 @@ module.exports = {
 	rules: {
 		'arrow-body-style': 'off',
 		'dot-notation': 'off',
+		'jest/no-conditional-expect': 'off',
+		'jest/valid-title': 'off',
```

Comment:
why are you turning this off?

Reply:
it's a new rule, but we are looping the test using protocol-spec in a lot of places